### ISSUE TITLE
is the json file necessary for NOT_strong training? 

### ISSUE THREAD

**Issue by xl-tang3** [iamalexkorotin/NeuralOptimalTransport#5]
Hello.
I got a task that transports the images from degreaded  domain to target domain (e.g., super-resolution), how to use this program to train a NOT map network?

**Comment by iamalexkorotin** [comment]
Json file just keeps the InceptionV3 neural network statistics (mean and covariance) of the test dataset. This is needed to faster compute the evaluation FID metric. You may create this json file for you own dataset by using [compute_stats.ipynb](https://github.com/iamalexkorotin/NeuralOptimalTransport/blob/main/stats/compute_stats.ipynb) notebook used with your own dataloader.

If you do not want to use FID metric, you can just comment out the parts "Loading data stats for testing" and "Computing FID" in the last cell  in [NOT_training_strong.ipynb](notebooks/NOT_training_strong.ipynb). The file should work without the neccesity to use this json.

For super-resolution, you may first upscale (e.g., bicubically), images from your first domain to the desired size, and then learn a transport map between these trivially upscaled (and hence low-detail) images and the detailed high-res images. The code should work well, you just need to change the dataloaders correspondingly.
